tests/foreman/installer/test_installer.py

```diff
@@ -1564,3 +1583,108 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_positive_capsule_installer_and_register():
+    """Verify the capsule installation and their registration with the satellite.
+
+    :id: efd03442-5a08-445d-b257-e4d346084379
+
+    steps:
+        1. Install the satellite.
+        2. Add all the required cdn and custom repositories in satellite to
+           install the capsule.
+        3. Create life-cycle environment,content view and activation key.
+        4. Subscribe the capsule with created activation key.
+        5. Run 'yum update -y' on capsule.
+        6. Run 'yum install -y satellite-capsule' on capsule.
+        7. Create a certificate on satellite for new installed capsule.
+        8. Copy capsule certificate from satellite to capsule.
+        9. Run the satellite-installer(copy the satellite-installer command from step7'th
+            generated output) command on capsule to integrate the capsule with satellite.
+        10. Check the newly added capsule is reflected in the satellite or not.
+        11. Check the capsule sync.
+
+
+    :expectedresults:
+
+        1. Capsule integrate successfully with satellite.
+        2. Capsule sync should be worked properly.
+
+    :CaseImportance: Critical
+
+    :CaseLevel: System
+
+    :CaseAutomation: NotAutomated
+    """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_positive_satellite_installer_logfile_check():
```

Comment:
It would be pretty easy to automate this using the existing fixture that will deploy a fresh satellite

Reply:
@mshriver In this PR, it is difficult to include because we get a lot of error and warning messages in various log files in which some of them are expected(due to the infra or pre-requirement) and some are valid issues, if we only check error messages and warning then this test case will always create noise, so I need some time to identify the common check and it is little bit big task so I think separate MR would be better.